src/kv/tables.rs

```diff
@@ -1066,7 +1066,7 @@ decl_table!(CallTraceSet => BlockNumber => CallTraceSetEntry);
 decl_table!(CallFromIndex => Vec<u8> => RoaringTreemap);
 decl_table!(CallToIndex => Vec<u8> => RoaringTreemap);
 decl_table!(BlockTransactionLookup => H256 => TruncateStart<BlockNumber>);
-decl_table!(Config => H256 => ChainConfig);
+decl_table!(Config => H256 => ChainSpec);
```

Comment:
why are we saving this to DB, and what's the key here?

Reply:
Key is genesis block hash - this is inherited from Erigon and Silkworm. Chainspec is saved to DB on genesis init in order to prevent accidentally misrunning the node with another chainspec.